### PR TITLE
Simplify "Complete workflow" with Docker meta action

### PR DESCRIPTION
**Pull request by crazy-max**
Use [Docker meta action](https://github.com/crazy-max/ghaction-docker-meta) to handle tags and labels based on GitHub actions events and Git metadata.

image

Signed-off-by: CrazyMax <crazy-max@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/example.yml [hunk 1]
```diff
@@ -25,34 +27,15 @@ jobs:
...
-          fi
-          echo ::set-output name=version::${VERSION}
-          echo ::set-output name=tags::${TAGS}
-          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+        name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: ${{ env.DOCKER_IMAGE }}
+          tag-sha: true
```
Comment (tonistiigi) [inline .github/workflows/example.yml:35]: I think it is hard to guess what exactly each of these options do by just looking at the example. Should at least add comments for every line. Or link to a separate example that uses this action for people who are ok with the learning curve to save some lines.

Comment (crazy-max) [reply .github/workflows/example.yml:35]: Done. I have simplified the workflow that should handle the most common cases.

Comment (tonistiigi) [reply .github/workflows/example.yml]: Comment all the options that are being shown (tag-edge etc) in upgrade. I'd also leave in an example of the old way as it is much flexible if there isn't an option that exactly matches. You can link to the right section in upgrade from the main readme.

Comment (crazy-max) [reply .github/workflows/example.yml]: Done

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**